lib/checker.js

```diff
@@ -45,6 +46,38 @@ Checker.prototype.checkFile = function(path) {
 };
 
 /**
+ * Extract JavaScript from file.
+ *
+ * @param {String} path
+ * @returns {Promise * Errors}
```

Comment:
I believe we typically do `@returns {Promise} descriptionHere` where the description can describe the resolved values. In this case it's an error list or null.